.circleci/config.yml

```diff
@@ -342,9 +342,11 @@ steps-mksnapshot-build: &steps-mksnapshot-build
         command: |
           cd src
           ninja -C out/native_mksnapshot v8:mksnapshot
+          ninja -C out/native_mksnapshot electron:electron_mksnapshot_zip
```

Comment:
If `electron:electron_mksnapshot_zip` depends on `v8:mksnapshot`, no reasons to build `v8:mksnapshot` first.

Reply:
it doesn't; that's the reason we're doing it this way; `v8:mksnapshot` isn't exposed in v8's visibility list. To depend on it that way, we'd have to patch v8's `BUILD.gn`